plugins/android_boot/android-boot.quirk

```diff
@@ -0,0 +1,16 @@
+[BLOCK]
+Plugin = android_boot
+
+# SHIFT6mq ABL A
+[DRIVE\UUID_c49183ed-aaec-9bf5-760a-66330fbcffc1&LABEL_abl-a]
+Flags = updatable,signed-payload
```

Comment:
This will only run for the supported GUIDs, but always be true where it runs right?  So perhaps these flags should just hardcode in the source in that case.

Reply:
Well depending on the previous bootloader stages, ABL might be signed or not.
I haven't seen much ABLs which are *not* signed, but in theory it is possible.

Updatable is just added here to only expose partitions which should be updatable for a given device, otherwise you get all partitions as devices in fwupd (which we really should avoid).